docs/Contributors_Guide/add_use_case.rst

```diff
@@ -120,6 +120,11 @@ Use Case Rules
   processed for analysis. Users can take an example and modify the run times
   to produce more output as desired.
 - No errors should result from running the use case.
+- The amount of required processing memory should not exceed the current limit 
```

Comment:
I think this item can be removed. The information about memory usage for running in automation is well documented below with your other additions. This information isn't as much a rule for developing the use case as it is a rule for adding it to the testing environment.